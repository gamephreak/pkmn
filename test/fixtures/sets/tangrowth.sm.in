Tangrowth||assaultvest|H|gigadrain,knockoff,powerwhip,earthquake|Sassy|248,,8,,252,||,30,30,,,|||,ice,
