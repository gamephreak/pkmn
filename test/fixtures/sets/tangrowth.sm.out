Tangrowth @ Assault Vest  
Ability: Regenerator  
EVs: 248 HP / 8 Def / 252 SpD  
Sassy Nature  
IVs: 30 Atk / 30 Def  
- Giga Drain  
- Knock Off  
- Power Whip  
- Earthquake  

